api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Extensions/ModelExtensions.cs

```diff
@@ -1301,7 +1302,8 @@ public static class ModelExtensions {
                 TriggerId = model.TriggerId,
                 SelectClauses = model.SelectClauses?
                     .Select(f => f.ToApiModel())
-                    .ToList()
+                    .ToList(),
+                PendingAlarm = model.PendingAlarm.ToApiModel()
```

Comment:
I suggest to pluralize this as PendingAlarms.

Reply:
As it could be multiple alarms per source node? Yeah, I see your point. However the follwowing declaration looks weird:

public PendingAlarmModel PendingAlarms { get; set; }

Reply:
Maybe the model should be pluralis as well, but it doesn't contain a collection.

Reply:
Yeah exactly. And not just that it can be, I think it will be the common case. You subscribe to a top node and then take everything from that. Maybe something like PendingAlarmsOptions?

Reply:
Yeah, we could try that.